### ISSUE TITLE
What happened?

### ISSUE THREAD

**Issue by Secret-chest** [C1710/blobmoji#158]
Is this dead? Should I fork?

**Comment by nonetrix** [comment]
Has emoji 15.1 been implemented?

**Comment by Secret-chest** [comment]
Silence.

**Comment by DavidBerdik** [comment]
@Secret-chest I see that you have forked this repo. Are you actively maintaining it?

**Comment by Secret-chest** [comment]
No. I want to see if there's something.

**Comment by DavidBerdik** [comment]
@Secret-chest Very well. Thanks for letting me know!

**Comment by Secret-chest** [comment]
> @Secret-chest Very well. Thanks for letting me know!

I might start maintaining because I'm making a Linux distro and none of the emoji fonts looks nice enough.

**Comment by DavidBerdik** [comment]
> I might start maintaining because I'm making a Linux distro and none of the emoji fonts looks nice enough.

Please do! I would love to have an updated version of this emoji set to use on Android!

**Comment by Secret-chest** [comment]
I will when I have time.

**Comment by DavidBerdik** [comment]
Great! I'll be on the lookout for it! 😁

**Comment by Secret-chest** [comment]
When I can, not now. Would you mind if I replace smiley emoji with the Android 6 version, or do I maintain two branches?

**Comment by jobukkit** [comment]
> When I can, not now. Would you mind if I replace smiley emoji with the Android 6 version, or do I maintain two branches?

I would not mind, the different poses made the blobs extra expressive.

**Comment by DavidBerdik** [comment]
> When I can, not now. Would you mind if I replace smiley emoji with the Android 6 version, or do I maintain two branches?

Yup! No problem! I understand life gets busy sometimes. I'm cool with bringing back the Android 6 version!

**Comment by nataliakeniganti** [comment]
Add Unicode 16.0

**Comment by Secret-chest** [comment]
I will try

**Comment by Secret-chest** [comment]
Understand I'm busy, though

**Comment by nataliakeniganti** [comment]
Used pleading face to eyebags
![emoji_u1fae9](https://github.com/user-attachments/assets/6807c22c-86dc-459a-894d-d11caef707ca)

**Comment by imaddictedtoemojis** [comment]
> Used pleading face to eyebags ![emoji_u1fae9](https://private-user-images.githubusercontent.com/141835174/386087173-6807c22c-86dc-459a-894d-d11caef707ca.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MDEyMTAsIm5iZiI6MTczNTYwMDkxMCwicGF0aCI6Ii8xNDE4MzUxNzQvMzg2MDg3MTczLTY4MDdjMjJjLTg2ZGMtNDU5YS04OTRkLWQxMWNhZWY3MDdjYS5zdmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIzMFQyMzIxNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OTZlM2QxYTVlNzJjNmU2NjRmMDYzMzEwYWMzMjAwN2I5YzczMDBjZjhhNjY2OGZhNjFlOWM1NDEyYmFiN2UzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fI_um8IsnOTjjVzl0pqLI8WJMZZRdByeGdAk3LOD6Ok)

i made mine, check https://github.com/C1710/blobmoji/issues/163. 
Unicode 17 >>> https://github.com/C1710/blobmoji/issues/164

**Comment by Secret-chest** [comment]
Really sorry, I want to fork this one day, but I have way more important things than an emoji font

**Comment by DavidBerdik** [comment]
I decided to spend some time exploring this evening and managed to get a compiled version of Blobmoji 16. I have attached the TTF file as well as the Magisk module for it below. GitHub does not allow the TTF file type to be uploaded to comments so it is contained in a zip file.

Before anyone gets too excited though, this build is nothing more than a merge of the blobmoji emojis with the emojis from Google's upstream repository. Essentially, all I have done is fill in the blanks with the missing emojis so that we'd be able to get a working font file that is compatible with Emoji 16. Because of this, the new emoijs that have been introduced since the last time this repo was updated are not "blobby." I am not an artist and have no interest in taking the time to update the necessary emojis to be blobby, but if someone else wants to do it and submit a pull request to [my repository](https://github.com/DavidBerdik/blobmoji-font) containing the necessary changes, I'd be happy to recompile the font with the updates and share it. It looks like @imaddictedtoemojis and/or @nataliakeniganti might be interested in taking this on, but I'll leave it up to them to decide. 🙂

I've installed this version of the font on my dev phone and as far as I can tell, it does not seem to have any issues, but of course, I would appreciate a second opinion.

Downloads:

- [Blobmoji-TTF.zip](https://github.com/user-attachments/files/19460579/Blobmoji.zip)
- [Blobmoji-Magisk.zip](https://github.com/user-attachments/files/19460589/Blobmoji-Magisk.zip)

**Comment by DavidBerdik** [comment]
I just made a proper release of Blobmoji 16 from my own repo. It took a bit more work than I had planned, but I decided to fork from Google's repo and merge in the emojis from this repository. If I am going to maintain it moving forward, I want to stay as close to using Google's build system as possible. This repo has deviated from that build system quite a bit which would make it more challenging for me.

If you just want the font files: https://github.com/DavidBerdik/blobmoji2/releases/tag/blobmoji-16r1

If you want a Magisk module with this update: https://github.com/DavidBerdik/blobmoji-magisk/releases/tag/16-r1
